parallel-trials/namespace-1/kustomization.yaml

```diff
@@ -3,6 +3,7 @@ kind: Kustomization
 
 resources:
 - ./namespace.yaml
-- ../../postgres/application
+# TODO: not sure how this has ever worked, but it definitely is incompatible with stormforge 2.0
+# - ../../postgres/application
```

Comment:
Did it need `--load-restrictor LoadRestrictionsNone` ?

Reply:
Or maybe it needs the full path (e.g. `github.com/thestormforge/examples/postgres/application`) like the Perf example?

Reply:
Kustomize can handle relative paths just fine. It's `kubectl` complaining that you cannot create that CRD against kube api. I think it's because we always generate an experiment from an application, and submit that to the api.

As for running parallel trials in multiple namespaces, I am not sure this above would have ever worked. Would it have?